src/ChunkMap.cpp

```diff
@@ -2593,6 +2593,34 @@ bool cChunkMap::ForEachChunkInRect(int a_MinChunkX, int a_MaxChunkX, int a_MinCh
 
 
 
+bool cChunkMap::ForEachLoadedChunk(cChunkDataCallback & a_Callback)
```

Comment:
I was not sure about this. At first i wanted to maintain same form like ForEachChunkInRect. However i'm not sure if it can fail? Is there logic to have it bool return?